tools/image-tag

```diff
@@ -5,9 +5,9 @@ set -o nounset
 set -o pipefail
 
 WIP=$(git diff --quiet || echo '-WIP')
-BRANCH=$(git rev-parse --abbrev-ref HEAD | sed s#/#-#g)
-SHA=$(git rev-parse --short=7 HEAD | head -c7)
+BRANCH=$(git rev-parse --abbrev-ref HEAD | sed 's#/#-#g')
+SHA=$(git rev-parse --short=7 HEAD)
 
-# If this is a tag, use it,                      otherwise branch-hash
```

Comment:
This indentation was on purpose, please leave it

Reply:
why need it? The comment in bash script seems no one to read.